src/main/java/core/basesyntax/strategy/CalculateStrategy.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandler;
+import core.basesyntax.service.impl.OperationHandlerBalance;
+import core.basesyntax.service.impl.OperationHandlerIn;
+import core.basesyntax.service.impl.OperationHandlerOut;
+import java.util.Map;
+
+public class CalculateStrategy {
+    private Map<FruitTransaction.Operation, OperationHandler> correspondenceTable = Map.of(
+            FruitTransaction.Operation.BALANCE, new OperationHandlerBalance(),
+            FruitTransaction.Operation.SUPPLY, new OperationHandlerIn(),
+            FruitTransaction.Operation.RETURN, new OperationHandlerIn(),
+            FruitTransaction.Operation.PURCHASE, new OperationHandlerOut());
```

Comment:
Oleh's comment not fixed, please create this map in Main method and pass as argument to constructor